### DISCUSSION TITLE
How do edit maid voice packs

### DISCUSSION THREAD

**Discussion by Revv1e** [TartaricAcid/TouhouLittleMaid#622] [Q&A]
Reading through the sound pack creation in here http://tlmwiki.cfpa.team/sound.html#creation-method I noticed there are different voice lines you can set for things like farming, torching and such, but when I check them in the game, they are always the same as the idle voices, how can I modify a sound pack in the tlm_custom_pack so that there is different voices for things like farming and such? Following the website guide isnt yielding any results :(

**Comment by Revv1e** [comment]
There are several actions that use the maid idle voices, is this a bug?

**Comment by TartaricAcid** [comment]
http://page.cfpa.team/TLM-Utils-Plugins/

Use this plugin to create a sound pack

We reused a lot of sound files
